lib/tasks/republish.rake

```diff
@@ -0,0 +1,8 @@
+namespace :republish do
+  desc "republish all guides"
+  task guides: :environment do
+    Guide.find_each do |guide|
+      GuideRepublisher.new(guide).republish
```

Comment:
Maybe some output here because the publisher is really slow. Going to merge it anyway though... nice to add in the future though.